From 77f6a709e86360f2a2f41b09ae3d069c65dcfd92 Mon Sep 17 00:00:00 2001
From: pobthebuilder <pobthebuilder@pobthebuilder.org>
Date: Mon, 22 Jan 2024 20:37:27 -0800
Subject: [PATCH] Fix opencl_blit_objects include stddef

xxd latest changed the type of the _len objects generated
with -i to be size_t, which requires stddef.h

---
 src/image/blit_src/create_hsaco_ascii_file.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/image/blit_src/create_hsaco_ascii_file.sh b/src/image/blit_src/create_hsaco_ascii_file.sh
index 3f9e51d..a0809aa 100755
--- a/src/image/blit_src/create_hsaco_ascii_file.sh
+++ b/src/image/blit_src/create_hsaco_ascii_file.sh
@@ -56,6 +56,8 @@ cat <<EOF
 //  This file is automatically generated during build process, don't modify it
 //==============================================================================

+#include <stddef.h>
+
 namespace rocr {
 namespace image {

--
2.41.0
